common/server_room.cpp

```diff
@@ -232,6 +234,18 @@ void Server_Room::say(const QString &userName, const QString &s, bool sendToIsl)
     event.set_name(userName.toStdString());
     event.set_message(s.toStdString());
     sendRoomEvent(prepareRoomEvent(event), sendToIsl);
+
```

Comment:
No I was correct.  So what is happening here is that a message is being sent by a user to the room.  The event above your noted line is the message that needs to make it to the room and the block of code below your note is the block that stores the message inside the PB message container and inserts it into the QList.  So skipping any portion of the code here would be a bad idea as you would either be not placing the event for the users message to make it to the room and since the history size starts at 0, skipping the lower block of code below the comment would never store a message in the history QList.